### PR TITLE
Awcc migration

### PR DESCRIPTION
**Pull request by rguichard**
new aw theme with aw colors set and logo

### CODE REVIEW

File: cours/intro-awcc.en.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
+ex Osones - <https://cloud-consulting.alterway.fr>
 
-![AWCC logo](images/logo-awcc.jpg){height="100px"}
+![AWCC logo](images/logo-awcc.jpg){height="70px"}
 
 - Copyright © 2014 - 2019 alter way Cloud Consulting
 - License: [Creative Commons BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/deed.fr)
 - Sources: <https://github.com/Osones/formations/>
-- HTML/PDF: <https://formation.alterway.fr/cloud/>
+- HTML/PDF: <https://osones.com/formations/>
```
Comment (rguichard) [inline cours/intro-awcc.en.md]: Should it be "https://cloud-consulting.alterway.fr/formations/" ?

File: cours/intro-awcc.fr.md [hunk 2]
```diff
@@ -2,14 +2,14 @@
...
+ex Osones - <https://cloud-consulting.alterway.fr>
 
-![Logo AWCC](images/logo-awcc.jpg){height="100px"}
+![Logo AWCC](images/logo-awcc.jpg){height="70px"}
 
 - Copyright © 2014 - 2019 alter way Cloud Consulting
 - Licence : [Creative Commons BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/deed.fr)
 - Sources : <https://github.com/Osones/formations/>
-- HTML/PDF : <https://formation.alterway.fr/cloud/>
+- HTML/PDF : <https://osones.com/formations/>
```
Comment (rguichard) [inline cours/intro-awcc.fr.md]: Same as the english version

File: cours/intro-awcc.en.md (hunk 1 above)
Comment (Adri2000) [reply cours/intro-awcc.en.md]: Nope, I don't think so. This is a corporate web page, with some links to some - potentially outdated - materials, whereas https://osones.com/formations/ is the exhaustive, up to date, list, of all up to date materials built by the CI.

File: cours/intro-awcc.fr.md (hunk 2 above)
Comment (Adri2000) [reply cours/intro-awcc.fr.md]: Same answer :)

File: cours/intro-awcc.en.md (hunk 1 above)
Comment (rguichard) [reply cours/intro-awcc.en.md]: Oh yes it points directly to the S3 bucket !

**Pull request merged**